src/views/LoginView.vue

```diff
@@ -1,35 +1,94 @@
 <template>
   <div>
     <main>
-      <img id="login-image" src="../assets/img/bier.png" alt="logo" />
+      <img
+          id="login-image"
+          src="../assets/img/bier.png"
```

Comment:
werkt dit ook als je build? Lijkt op relative routing en dan moet je @ gebruiken. Zie de vite config enzo.

Reply:
This should be fixed now in commit https://github.com/GEWIS/sudosos-frontend-vue3/pull/31/commits/c0cd5afae61160be4c481f76a38752281f930aa6